config.yml

```diff
@@ -0,0 +1,7 @@
+---
+host: localhost
```

Comment:
Can we move this to an `examples` directory?

Reply:
yes!